xbmc/pvr/channels/PVRChannelGroups.h

```diff
@@ -194,6 +194,12 @@ namespace PVR
      */
     bool Update(bool bChannelsOnly = false);
 
+    /*!
+     * @brief Update the channel numbers across the groups after using channel manager
```

Comment:
Not a good comment, imo. I would only explain what it does, not the (current) callers. I bet you will forget to update this comment if you ever change the channel manager code no longer to call this method.